src/devices/machine/nscsi_cd.cpp

```diff
@@ -239,8 +244,14 @@ void nscsi_cdrom_device::scsi_command()
 		break;
 	}
 
+	case SC_PREVENT_ALLOW_MEDIUM_REMOVAL:
+		// TODO: support eject prevention
+		LOG("command PREVENT ALLOW MEDIUM REMOVAL\n");
+		scsi_status_complete(SS_GOOD);
+		break;
+
 	default:
-		fprintf(stderr, "scsi %02x\n", scsi_cmdbuf[0]);
+		LOG("unhandled command %02x\n", scsi_cmdbuf[0]);
```

Comment:
Perhaps this one should be straight `logerror` so unhandled commands are always logged, not just when verbose is on?